Remember The Date/Framework/ZendeskSDK.framework/Headers/ZDKUiConfiguration.h

```diff
@@ -1,18 +1,10 @@
-/*
- *
- *  ZDKUiConfiguration.h
- *  ZendeskSDK
- *
- *  Created by Zendesk on 24/09/2014.
- *
- *  Copyright (c) 2014 Zendesk. All rights reserved.
- *
- *  By downloading or using the Zendesk Mobile SDK, You agree to the Zendesk Master
- *  Subscription Agreement https://www.zendesk.com/company/customers-partners/#master-subscription-agreement and Application Developer and API License
- *  Agreement https://www.zendesk.com/company/customers-partners/#application-developer-api-license-agreement and
- *  acknowledge that such terms govern Your use of and access to the Mobile SDK.
- *
- */
+//
+//  ZDKUiConfiguration.h
+//  ZendeskSDK
+//
+//  Created by Steven Diviney on 12/06/2018.
+//  Copyright © 2018 Zendesk. All rights reserved.
+//
```

Comment:
This header file should not have changed! I think it's an indicator we've done something wrong